Update experiments doc (#983)
rlworkgroup/garage: docs/user/experiments.rst

.. code-block:: python

    from garage.experiment import run_experiment
    from garage.np.baselines import LinearFeatureBaseline
    from garage.tf.algos import TRPO
    from garage.tf.envs import TfEnv
    from garage.tf.experiment import LocalTFRunner

>     from garage.tf.policies import

Reviewer: Looks like the import got missed
```suggestion
    from garage.tf.policies import CategoricalMLPPolicy
```

    def run_task(snapshot_config, *_):

>         """Run task."""

Reviewer: I think the existing docstring was fine
```suggestion
        """Wrap TRPO training task in the run_task function."""
```

Running Experiments with TensorFlow and GPU
===========================================

> Using `tensorflow-gpu`, TensorFlow has GPU enabled by default. Set the flags `force_cpu` to True to disable GPU, as shown in the code below:

Reviewer: Maybe a nitpick, but I suggest this to be consistent with installation docs
```suggestion
If you installed the `garage[gpu]` extra package, TensorFlow will use GPU by default. To disable GPU, set the flag `force_cpu` to True, as shown in the code below:
```

Author: Thanks for suggestions! Updated.
